src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -793,7 +793,7 @@ class SpatialPooler : public Serializable
   // NOT part of the public API
 
 
-  void boostOverlaps_(const vector<UInt> &overlaps, vector<Real> &boostedOverlaps) const;
+  virtual void boostOverlaps_(const vector<UInt> &overlaps, vector<Real> &boostedOverlaps) const;
```

Comment:
can overlaps be SDR, each column has it's boostedOverlap ?

Reply:
No.  SDR's hold binary activations.  Overlaps contains integers: the number active presynapses per segment.